cmd/agent/main.go

```diff
@@ -41,7 +41,9 @@ var (
 )
 
 const (
-	Token = "X-API-Tunnel-Token"
+	Token                        = "X-API-Tunnel-Token"
+	RegenerateKubeletCertificate = "Regenerate-Kubelet-Certificate"
```

Comment:
nit: a certificate isn't being regenerated if it's a new node on the first run, but I think this is not really important as long as we're consistent.

Can you add a quick comment here just stating what this header does?